lib/screens/home/home_event.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/cupertino.dart';
+import 'package:meta/meta.dart';
+import 'package:pub_client/pub_client.dart';
+
+@immutable
+abstract class HomeEvent {}
```

Comment:
are you intentionally not using `Equatable`?

Reply:
Nope. It's not a default and I don't have a need to use it for the time being. This was all thrown together in about an hour. I plan on adding what's needed when I see it. Is it needed here?

Reply:
@felangel

Reply:
It's needed if you want to be able to compare different instances of events and states which comes in handy if you want to make optimizations to avoid unnecessary rebuilds.